js/debugLog.js

```diff
@@ -41,7 +41,7 @@
     });
 
     var MAX_MESSAGES = 1000;
-    var PHONE_REGEX = /\+\d{7,12}(\d{3})/g;
```

Comment:
You are correct! The android logs do show only two digits. But have you tested this? Shouldn't the other two numbers in the regex be incremented?